backend/offers_web.py

```diff
@@ -32,9 +32,12 @@ def on_get(self, req, resp):
         """Returns all offers available"""
         limit, page = retrieve_paging_params(req.params, MAX_OFFERS)
 
-        try:
-            show_all = bool(req.params.get('show_all', False))
-        except ValueError as e:
+        show_all_str = req.params.get('show_all', 'false')
```

Comment:
Would be better to wrap this logic in a `cast_to_bool` or `coerce_bool`, we're going to reuse that a lot.